From 72cbe7bed77a18635a5688f9affa341ceb06a576 Mon Sep 17 00:00:00 2001
From: Peleg Vigodny <pvigodny@habana.ai>
Date: Tue, 31 Jan 2023 17:07:49 +0200
Subject: [PATCH] [SW-0] shorten hl version

workaround for build error:

"5.10.18-hl-gaudi2-1.9.0-rc-fw-38.1.99-3-gfdca9d57e985-dirty-sec-2"
exceeds 64 characters

Change-Id: I31abaffff934289c655cdec009b92e334e5dc4e7
Signed-off-by: Peleg Vigodny <pvigodny@habana.ai>
---

diff --git a/package/version-hooks/version-hooks.mk b/package/version-hooks/version-hooks.mk
index b727f7c..f9ffbd2 100644
--- a/package/version-hooks/version-hooks.mk
+++ b/package/version-hooks/version-hooks.mk
@@ -48,7 +48,7 @@
 		fi; \
 		CUR_VER+="fw-"; \
 		CUR_VER+="$${GIT_VER/*-fw-/}-"; \
-		CUR_VER+="sec-$(HL_SEC_FW_VERSION)"; \
+		CUR_VER+="s-$(HL_SEC_FW_VERSION)"; \
 		echo -n ">>> ${EFFECTIVE_PACKAGE_NAME} setting localversion to: "; \
 		mkdir -p $(@D)/"${SUB_PACKAGE_DIR}"; \
 		echo -n "-$${CUR_VER}" | tee $(@D)/"${SUB_PACKAGE_DIR}"/localversion-habanalabs; \
